code/modules/research/designs/AI_module_designs.dm

```diff
@@ -153,3 +153,165 @@
 	build_path = /obj/item/ai_module/core/full/custom
 	category = list("AI Modules")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/board/dungeon_master_module
+	name = "Core Module Design (Dungeon Master)"
+	desc = "Allows for the construction of a Dungeon Master AI Core Module."
+	id = "corporate_module"
```

Comment:
i think you forgot to change this, which is leading to a failed unit test on your checks

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/66854/commits/d70169df6529e124236033a13bad3de4cd119b58 thanks!